src/puptoo/process.py

```diff
@@ -126,6 +126,9 @@ def system_profile(
         profile["sap_system"] = True
         sids = {sap.sid(instance) for instance in sap.local_instances}
         profile["sap_sids"] = sorted(list(sids))
+        inst = sap.local_instances[0]
```

Comment:
to me it seems potentially misleading to list the sap_ids of all of them, but then have the instance numbers/version of only one,

would i be sensible to insert a mapping of instance number to version ,l or maybe the sap_sid of the instance

Reply:
I double checked with the product managers and subject matter expert, and we're going to move forward as is at this time, but thank you for pointing it out and we will give it more thought in the future if there's a use case for a different structure here.